docs/reference/commandline/rm.md

```diff
@@ -69,7 +69,7 @@ The main process inside the container referenced under the link `redis` will rec
 ### Remove all stopped containers
 
 ```bash
-$ docker rm $(docker ps -a -q)
+$ docker rm (docker ps -a -f "status=exited" -q)
 ```
 
 This command will delete all stopped containers. The command
```

Comment:
It's actually explained below that the `docker ps -a -q` returns all containers, but the `docker rm` will not remove running containers in the list.

Reply:
But still it is trying to rm them. Do you think we should leave it like it is?

Reply:
Well good point is it documented, that the running container won't be deleted. But still those messages are like an unexpected behaviour. For people, which are new to docker this is confusing.

Reply:
Well good point it is documented, that the running container won't be deleted. But still those messages are like an unexpected behaviour. For people, which are new to docker this is confusing.